### ISSUE TITLE
Spoilery w komentarzach na mikro nie otwierają się

### ISSUE THREAD

**Issue by dobrzecisiewydaje** [CodoDodo/Zakop#67]
Jak w tytule. Czy ktoś jeszcze ma ten problem?

**Comment by CodoDodo** [comment]
@dobrzecisiewydaje Tutaj wszystko w normie, w każdym wątku jest to samo?

**Comment by dobrzecisiewydaje** [comment]
Jak na razie w każdym napotkanym. Sprawdzam dalej!

**Comment by CodoDodo** [comment]
@dobrzecisiewydaje Bardzo dziwne, masz ostatnią wersję aplikacji?

**Comment by dobrzecisiewydaje** [comment]
Tak, ale przeinstalowałam na nowo i problem zniknął. Musiał być jakiś glitch z mojej strony. Do zamknięcia :)
